### PR TITLE
Replace vendor-specific examples and links

### PR DESCRIPTION
**Pull request by atlaskerr**
This PR replaces Docker manifest examples with the OCI. format.

Fixes issue #47

### CODE REVIEW

File: spec.md [hunk 1]
```diff
@@ -1450,17 +1461,23 @@ Docker-Content-Digest: <digest>
 Content-Type: <media type of manifest>
 
 {
-   "name": <name>,
-   "tag": <tag>,
-   "fsLayers": [
+   "annotations": {
```
Comment (mikebrow) [inline spec.md]: ditto...

**Comment by atlaskerr** [comment]
@mikebrow Thanks for looking over my work! When I was working on this last night, I realized that a bunch of external links to moby should be refactored to link to the image spec. I'll mark this as a work-in-progress.

File: spec.md [hunk 2]
```diff
@@ -660,20 +666,25 @@ An image can be pushed using the following request format:
...
-             "blobSum": <digest>
-          },
-          ...
-        ]
-       ],
-       "history": <v1 images>,
-       "signature": <JWS>,
-       ...
-    }
+	{
```
[the remark this answers is not in the document]
Comment (stevvooe) [reply spec.md]: Nah, leave the examples in place, as they represent a full http request.

File: spec.md [hunk 3]
```diff
@@ -291,10 +291,8 @@ For reference, the relevant manifest fields for the registry are the following:
 |-----------|------------------------------------------------|
 | name      | The name of the image.                         |
 | tag       | The tag for this version of the image.         |
-| fsLayers  | A list of layer descriptors (including digest) |
-| signature | A JWS used to verify the manifest content      |
 
-For more information about the manifest format, please see [moby/moby#8093](https://github.com/moby/moby/issues/8093).
+For more details on the manifest formats and their content types, refer to the OCI Image Specification's [manifest property description](https://github.com/opencontainers/image-spec/blob/master/manifest.md#image-manifest-property-descriptions).
```
Comment (mikebrow) [inline spec.md]: The description above in lines 288 -293 does not fit with this line.

**Comment by atlaskerr** [comment]
@opencontainers/distribution-spec-maintainers PTAL

**Comment by vbatts** [comment]
LGTM

**Pull request merged**